### PR TITLE
aws-rotate-key: 1.0.0 -> 1.0.3

### PR DESCRIPTION
**Pull request by mbode**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/admin/aws-rotate-key/default.nix [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+  version = "1.0.2";
 
   goPackagePath = "github.com/Fullscreen/aws-rotate-key";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "Fullscreen";
     repo = "aws-rotate-key";
-    sha256 = "13q7rns65cj8b4i0s75dbswijpra9z74b462zribwfjdm29by5k1";
+    sha256 = "0pcdnp1i88wxbxjj9cwpxasa785wc1nd6iy54sb12i95alxc4r3ml";
```
Comment (kalbasit) [inline pkgs/tools/admin/aws-rotate-key/default.nix]: you seem to have an extra `l` at the end of the sha256, the correct sha is `0pcdnp1i88wxbxjj9cwpxasa785wc1nd6iy54sb12i95alxc4r3m`

**Review by kalbasit — CHANGES_REQUESTED** [review]

File: pkgs/tools/admin/aws-rotate-key/default.nix (hunk 1 above)
Comment (mbode) [reply pkgs/tools/admin/aws-rotate-key/default.nix]: whoops, sorry about that!

**Comment by timokau** [comment]
@GrahamcOfBorg build aws_rotate_key

**Comment by kalbasit** [comment]
@GrahamcOfBorg build aws-rotate-key

**Comment by timokau** [comment]
Can you please fix the package name in the commit message?

**Comment by c0bw3b** [comment]
Apparently upstream already published 1.0.3 to fix the version reported by the tool.
@mbode would you be willing to make this PR about 1.0.0 -> 1.0.3 please?

**Comment by mbode** [comment]
@c0bw3b Thanks for noticing!

**Comment by c0bw3b** [comment]
```go
$ ./result-bin/bin/aws-rotate-key --version
1.0.3
```

**Review by c0bw3b — APPROVED** [review]

**Pull request merged**